src/components/Organisms/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from 'react';
+import './NavBarStyles.scss';
+import { NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTicketAlt, faBars } from '@fortawesome/free-solid-svg-icons';
+import SearchBar from '../../Atoms/SearchBar';
+
+function NavBar() {
+  const [collapse, setCollapse] = useState();
+  const [toggleClass, setToggleClass] = useState('');
+  const [toggleAria, setToggleAria] = useState('');
+  const [navClass, setNavClass] = useState('');
+
+  const updateToggleClass = (isActive) => {
+    if (isActive) {
+      setToggleClass('navbar-toggler collapsed');
+      setToggleAria('false');
+      setNavClass('navbar-collapse collapse');
+      return;
+    }
+    setToggleClass('navbar-toggler');
+    setToggleAria('true');
+    setNavClass('navbar-collapse collapse show');
+  };
+
+  useEffect(() => {
+    const initialToggleValue = true;
+    setCollapse(initialToggleValue);
+    updateToggleClass(initialToggleValue);
```

Comment:
`setCollapse` and `updateToggleClass` should probably be combined into a single method since they are always called together (being picky here).